lib/filewatch/ruby_fnv.rb

```diff
@@ -0,0 +1,63 @@
+# fnv code extracted and modified from https://github.com/jakedouglas/fnv-ruby
```

Comment:
@jsvd - do you mean document in this file or in the repo somewhere? Beats and us are discussing various other mechanisms - there may be another refactor with pluggable schemes